src/doc-utils.js

```diff
@@ -21,6 +21,10 @@ function traverseDoc(doc, onEnter, onExit) {
       if (doc.flatContents) {
         traverseDocRec(doc.flatContents);
       }
+    } else if (doc.type === "group" && doc.expandedStates) {
+      if (shouldTraverseConditionalGroups) {
+        doc.expandedStates.forEach(traverseDocRec);
+      }
```

Comment:
I don't call `traverseDocRec(doc.contents);` if we're not in `shouldTraverseConditionalGroups`